server/src/__tests__/donatedItem.test.ts

```diff
@@ -16,38 +16,48 @@ describe('DonatedItem API Tests', () => {
         // Setup mock responses
         mockPrismaClient.program.findUnique.mockResolvedValue(newProgram);
         mockPrismaClient.donor.findUnique.mockResolvedValue(newDonor);
-        mockPrismaClient.donatedItem.create.mockResolvedValue({ id: 1, ...newItem });
-        mockPrismaClient.donatedItem.update.mockResolvedValue({ id: 1, ...updateItem });
+        mockPrismaClient.donatedItem.create.mockResolvedValue({
+            id: 1,
+            ...newItem,
+        });
+        mockPrismaClient.donatedItem.update.mockResolvedValue({
+            id: 1,
+            ...updateItem,
+        });
     });
 
     // Test POST /donatedItem
     it('validates Program name and Donor email before creating a donated item', async () => {
         const response = await request(app).post('/donatedItem').send(newItem);
         expect(response.status).toBe(201);
         expect(mockPrismaClient.donatedItem.create).toHaveBeenCalled();
-    
```

Comment:
Consider creating a helper function for validation to avoid code duplication in POST and PUT routes.